comms/discovery/the_graph/discovery_nodes.go

```diff
@@ -0,0 +1,102 @@
+package the_graph
```

Comment:
nit... isnt go convention for package names all lowercase, no underscores/camelcase/etc?

Reply:
yeah... gonna change that package name soon to better support a hard coded list  or alternate sources... but for now just gonna merge this PR and do that as follow on.